platform-sdk/swirlds-config-impl/src/main/java/com/swirlds/config/impl/internal/ConverterService.java

```diff
@@ -128,6 +127,20 @@ <T> T convert(@Nullable final String value, @NonNull final Class<T> targetClass)
             return (T) value;
         }
         final ConfigConverter<T> converter = (ConfigConverter<T>) converters.get(targetClass);
+
+        if (converter == null && targetClass.isEnum()) {
+            // FUTURE WORK: once logging is added to this module, log a warning here
+            // ("No converter defined for type '" + targetClass + "'. Converting using backup enum converter.");
+            try {
+                return (T) Enum.valueOf((Class<Enum>) targetClass, value);
+            } catch (final IllegalArgumentException e) {
+                throw new IllegalArgumentException(
+                        "Can not convert value '%s' of Enum '%s' by default. Please add a custom config converter."
+                                .formatted(value, targetClass),
+                        e);
+            }
+        }
```

Comment:
This is cool, the only problem that I see is that after this change, the method
 getConverterForType(@NonNull final Class<T> valueType) 
 would work inconsistently compared with #convert. 

Regardless I don't think that the change required to address my comment is in the scope of the current task.  What do you think @hendrikebbers should we create a follow-up task to make those two behaviors consistent on our side?